src/server/scripts/Outland/CoilfangReservoir/TheUnderbog/boss_the_black_stalker.cpp

```diff
@@ -99,25 +99,25 @@ struct boss_the_black_stalker : public BossAI
                         EnterEvadeMode();
```

Comment:
I don't know how to feel about this old boundary check inside EventMap, might as well update it to be checked from InstanceScript directly

Reply:
We need Leash to be implemented, it's original thingy

Reply:
well, I meant encounter boundary in our own terms where we can choose what type of, radius from point, rectangle, polygon, etc.

Reply:
That boundary system has nothing close to original Leash field

Reply:
It's just radius boundary, yeah
Well, many DB things on retail can be overriden in Lua